run-qa-checks

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+openwisp-qa-check \
+
+    --skip-isort \
+    --skip-flake8 \
+    --skip-checkendline \
+    --skip-checkcommit \
+    --skip-black \
```

Comment:
Why are you doing this? Please read https://github.com/openwisp/openwisp-utils#quality-assurance-checks

Reply:
I don't think we need this script to run a single command. Shouldn't we write `openwisp-qa-format` in ci directly. 🤔

Reply:
Yes, we can ignore it. Its not a big issue